src/components/Menu/Menu.jsx

```diff
@@ -1,9 +1,31 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { actions, categoriesSelectors } from '../../store';
 import css from './styles.module.css';
 
+
 export function Menu() {
+    const dispatch = useDispatch();
+    const categoriesItem = useSelector(categoriesSelectors.getCategories);
```

Comment:
```suggestion
    const categories = useSelector(categoriesSelectors.getCategories);
```

Т.к. твой селектор уже достает поле categories:
```export const getCategories = (state) => getCategoriesOriginal(state).categories;```